### PR TITLE
TV Content Page Updates

### PR DESCRIPTION
**Pull request by bryantgillespie**
- Adds feedback widget for like, dislike, love
- Adds recommendations to other episodes or content for episodes detail pages
- Adds transcript to episode pages and JSON LD data

### CODE REVIEW

File: pages/tv/[show]/[episode].vue [hunk 1]
```diff
@@ -84,19 +119,66 @@ const { data: next } = await useAsyncData(
...
+
+useSchemaOrg([
+	defineVideo({
+		name: unref(episode).title,
+		description: unref(episode).description,
+		thumbnailUrl: `${tvUrl}/assets/${unref(episode).tile}`,
+		uploadDate: unref(episode).published,
+		duration: unref(episode).duration,
+		embedUrl: `${baseUrl}/tv/${route.params.show}/${route.params.episode}`,
+		transcript: unref(episode).video_transcript_text,
```
Comment (paescuj) [inline pages/tv/[show]/[episode].vue:179]: Not sure, but should we perhaps truncate the transcript to something like 200 chars as they can become quite long?

**Review by paescuj — COMMENTED** [review]
Do we need the top margin on the notice element? The gap feels a bit odd


---

Should we add additional dividers on mobile devices? I'm thinking about at least one between the "About" and "Meta (Resources, People, ...) sections:

File: pages/tv/[show]/[episode].vue (hunk 1 above)
Comment (bryantgillespie) [reply pages/tv/[show]/[episode].vue:179]: One of the major reasons to add the transcripts at all was for SEO purposes to help improve visibility and ranking of these pages. So I don't want to shorten them, even if they run a bit long.

Comment (paescuj) [reply pages/tv/[show]/[episode].vue:179]: Okay, sounds good!

Comment (phazonoverload) [reply pages/tv/[show]/[episode].vue:179]: Yeah, I back Bryant here - it'll be important

Comment (paescuj) [reply pages/tv/[show]/[episode].vue:179]: The SEO experts have spoken 😄🧑‍⚖️

**Comment by phazonoverload** [comment]
While I originally said I didn't want to surface the transcript in the UI, I like this tabbed approach. I'd suggest we also move the people, resources, and maybe even 'show details' into their own tabs.

**Comment by bryantgillespie** [comment]
> I'd suggest we also move the people, resources, and maybe even 'show details' into their own tabs.

People: let's keep as is. I like those are always visible. Especially for external guests.
Resources: 💯
Show Details: What would this cover?

**Review by paescuj — APPROVED** [review]
LGTM 🚀

**Pull request merged**